samples/msal-angular-v2-samples/README.md

```diff
@@ -1,11 +1,11 @@
 # MSAL Angular v2 Samples
 
-**`@azure/msal-angular` is now available for public preview. Please see the [`@azure/msal-angular` README](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular) for information on installation and changes.** 
+**`@azure/msal-angular` is now available for public preview. Please see the [`@azure/msal-angular` README](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular) for information on installation and changes.**
```

Comment:
Please remove this section since msal-angular v2 has been GA'd

Reply:
done!